src/components/ProductCard.tsx

```diff
@@ -0,0 +1,38 @@
+import useToggleCartMutation from '@/hooks/queries/useToggleCartMutation';
+import { Product } from '@/models/product.model';
```

Comment:
```suggestion
import type { Product } from '@/models/product.model';
```

번들 크기에 대해서 신경쓰신 부분이 있어서 코멘트를 남기자면
타입 관련 모듈이 빌드 결과물에 포함되지 않도록 신경써볼 수도 있을 것 같아요 🤔

Reply:
https://typescript-eslint.io/rules/consistent-type-imports/

오~ eslint로 해결할 수 있을 것 같아 검색해보니 이런 룰이 있네요! 룰 적용해보았습니다.

Reply:
https://github.com/next-step/react-shopping-cart/pull/38/commits/ae1a4485edb01cc94ae693b869fcbecab03ae410